### DISCUSSION TITLE
webview2- reopen PDF at last viewed location

### DISCUSSION THREAD

**Discussion by ookuboToshihiro** [MicrosoftEdge/WebView2Feedback#3980] [Q&A]
I am using webview2 to display the pdf.
I want to get the scroll position of the currently displayed pdf.

document.documentElement.scrollTop and window.scrollX
cannot be obtained.

How can I get it?

**Comment by vickiez** [comment]
Hi @ookuboToshihiro, can you share more about the scenario you are trying to accomplish? Are you trying to reopen the PDF at the last viewed location?

**Comment by ookuboToshihiro** [comment]
I'm creating an application with webview2 in VisualStudio.
I want to save the scroll position when displaying a pdf.
(I want to display at the same position when reopen pdf）


The following code can not get the scroll position.
How can I get it?

----------------------------------------------------------------------------------------------
 private async void button1_Click(object sender, EventArgs e)
{
    var js = new System.Text.StringBuilder();
    js.AppendLine("alert('scroll(document.documentElement.scrollLeft,document.documentElement.scrollTop) 　X：' + document.documentElement.scrollLeft + '　Y：' + document.documentElement.scrollTop);");
    js.AppendLine("alert('scroll(window.scrollX,Y)　X：' + window.scrollX + ' Y：' + window.scrollY );");
    js.AppendLine("alert('embed count = ' + document.getElementsByTagName('embed').length);");
    js.AppendLine("alert('scrollTop = '   + document.getElementsByTagName('embed').scrollTop);");

    await webView21.ExecuteScriptAsync(js.ToString());
}
--------------------------------------------------------------------------------------------------
![scroll-NG](https://user-images.githubusercontent.com/7483392/186571366-c28df90f-5e8f-4957-80fd-429693e64382.jpg)

**Comment by ookuboToshihiro** [comment]
The above PDF is read as follows.

webView21.CoreWebView2.Navigate("C:\\Users\\user\\Desktop\\abc.pdf");
